app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,70 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?
+      erb :'tweets/tweets'
+    else
+      redirect "/login"
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'tweets/create_tweet'
+    else
+      redirect "/login"
+    end
+  end
+
+  get '/tweets/:id' do
+    if logged_in?
+      @tweet = Tweet.find(params[:id])
+      erb :'tweets/show_tweet'
+    else
+      redirect "/login"
+    end
+  end
+
+  get '/tweets/:id/edit' do
+    if logged_in?
+      @tweet = Tweet.find(params[:id])
+      if @tweet.user == current_user
+        erb :'tweets/edit_tweet'
+      else
+        redirect "/tweets/#{@tweet.id}"
+      end
+    else
+      redirect "/login"
+    end
+  end
+
+  post '/tweets' do
+    redirect "/tweets/new" if params[:content].empty?
+
+    @tweet = Tweet.create(content: params[:content], user_id: current_user.id)
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.